Solutions/AristaAwakeSecurity/Analytic Rules/ModelMatchesWithMultipleDestinationsByDevice.yaml

```diff
@@ -0,0 +1,62 @@
+id: dfa3ec92-bdae-410f-b675-fe1814e4d43e
+name: Awake Security - Model Matches With Multiple Destinations By Device
+description: This query searches for devices with multiple possibly malicious destinations.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AristaAwakeSecurity
+    dataTypes:
+      - AwakeSecurity
+query: |
+  CommonSecurityLog | where DeviceVendor == "Arista Networks" and DeviceProduct == "Awake Security"
+  | summarize Models=make_set(Activity), ASPMatchURLs=make_set(DeviceCustomString2), SourceIPs=make_set(SourceIP),
+    DestinationIPs=make_set(DestinationIP), ModelMatchCount=sum(EventCount), MaxSeverity=max(toint(LogSeverity)) by SourceHostName
+  | where array_length(DestinationIPs) > 1
+queryFrequency: 1h
```

Comment:
The structure of the template should be fixed. queryFrequency, queryPeriod, triggerOperator, triggerThreshold, tactics, relevantTechniques should ideally come before the query.